apps/website/src/pages/docs/inputwithdecorations.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Input with decorations
+description: Input component with various additional decorations
+layout: ./_layout.astro
+propsPath: '@itwin/itwinui-react/esm/core/utils/components/InputFlexContainer.d.ts'
+thumbnail: #TODO
```

Comment:
this one could use same thumbnail as input for now?

Reply:
Using